packages/groqd-legacy/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.15.13
```

Comment:
So I'm pretty sure this will not get deployed properly.
It's currently renamed to `groqd-legacy`